README.md

```diff
@@ -2,6 +2,49 @@
 
 > Rust Crockford encoding for u64
 
-There is another Crockford Base32 library available for Rust; I just thought it was incredibly annoying to use, because it doesn't allow you to just plug in a `u64` and get back an encoded value. Rather than take a dependency on the existing library, this library is a marginally faithful port of a C# library that performs the same job.
+This is not the only such library for Rust; I just thought the other one was annoying to use, primarily because it does not simply let you convert a `u64` to a `String` and vice versa. One would imagine the thing to do, then, would be to write a wrapper to make the other library more convenient--but, no. I'm not a big believer in dependencies. As a result, this library is a marginally faithful port of a C# library that performs the same job.
```

Comment:
>  it does not simply let you convert a `u64` to a `String` and vice versa

Why is `u64` special? Is encoding a 64 bit unsigned integer a common use case for crockford base32? Maybe the readme should explain the motivation behind making this library focused on `u64`.

Reply:
`u64` just matches the use case most closely, I think. If I understand right (the website does not explicitly mention this), you can't encode a negative number. The most common use case (in fact, the use case I built this for) is shortening numeric database identifiers, so something "big enough" for that job is, likewise, appropriate.

I thought about going with `Into<u64>` or something. I dunno. /shrug

Good point re: readme.